src/Test.UnitTests.Sarif/Map/LongArrayDeltaConverterTests.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Map
+{
+    public class LongArrayDeltaConverterTests
+    {
+        internal class Container
+        {
+            [JsonConverter(typeof(LongArrayDeltaConverter))]
+            public List<long> Values;
+        }
+
+        [Fact]
+        public void LongArrayDeltaConverter_Basics()
+        {
+            RoundTrip(null);
+            RoundTrip(new long[] { });
+            RoundTrip(new long[] { -1 });
+            RoundTrip(new long[] { 0, 0, 0, 0, 0 });
+            RoundTrip(new long[] { 100, 101, 102, 103, 104, 105 });
+            RoundTrip(new long[] { 100, -100, 200, -200, 300, -300 });
```

Comment:
>RoundTrip [](start = 12, length = 9)

add a test that incorporates long max and min values, perhaps?

Reply:
Added, along with comments warning about the real range limits.